packages/search/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
+
+- Add search modes
+- Add "when-typing" and "on-enter" search mode
```

Comment:
Still can be shortened to just one line. Currently both entries say the same thing: search modes were added.